prebid-mobile/code-integration-ios.md

```diff
@@ -45,22 +45,33 @@ Register Prebid Mobile ad units as early as possible in the application's lifecy
 
 Note: As of the Prebid Mobile SDK Version 0.2.0, the existing registerAPIs listed below are deprecated:
 
-```[PrebidMobile registerAdUnits:@[adUnit1] withAccountId:@"YOUR-ACCOUNT-ID-HERE"];
+```
+[PrebidMobile registerAdUnits:@[adUnit1] withAccountId:@"YOUR-ACCOUNT-ID-HERE"];
 [PrebidMobile registerAdUnits:@[adUnit1, adUnit2] withAccountId:kAccountId andPrimaryAdServer:PBPrimaryAdServerDFP];
 ```
 
-We recommend doing this in the `didFinishLaunchingWithOptions` method in `AppDelegate.m` using the following steps as shown in the code sample below:
+
+Note: The existing registerAPIs listed below are deprecated.
+
+```
+[PrebidMobile registerAdUnits:@[adUnit1] withAccountId:@"YOUR-ACCOUNT-ID-HERE"];
+[PrebidMobile registerAdUnits:@[adUnit1, adUnit2] withAccountId:kAccountId andPrimaryAdServer:PBPrimaryAdServerDFP];
+```
+
+We recommend doing this in the `didFinishLaunchingWithOptions` method in `AppDelegate.m` using the following steps, as shown in the code sample below:
 
 1. Create the ad units with ad unit ids and add sizes for banner ad units.
-2. Add a server side configuration for each ad unit to Prebid Server Adapter.
-3. Set the targeting parameters for the ad units. (Optional)
-4. Set the primary adserver for the bid to either DFP or MoPub (the primary ad server is necessary to determine the caching mechanism).
-5. Set the host for the bid. Supported hosts are AppNexus and Rubicon.
+2. Add a server-side configuration for each ad unit to Prebid Server Adapter.
+3. Set targeting parameters for the ad units. (Optional)
+4. Set the primary adserver for the bid to either DFP or MoPub. (Primary ad server is necessary to determine the caching mechanism.)
+5. Set the Host for the bid to AppNexus or Rubicon.
 6. Register the ad units with the adapter to start the bid fetching process.
 
+
 Embed the ad unit registration in a try-catch block to catch all the exceptions (if any) thrown by the SDK.
 
-```objc
+```
+objc
```

Comment:
This looks to be causing the word "objc" to appear at the top of this and subsequent code snippets.